src/lib/slate-suggestions-dist/index.js

```diff
@@ -28,9 +29,19 @@ function getAllTextExceptIgnoredTypes(node, typesToIgnore) {
 function matchTrigger(state, trigger, typesToIgnore) {
     // Get the first block
     const currentNode = state.blocks.first();
+    const relevantSelection = {
+        anchorKey: currentNode.nodes.get(0).key,
```

Comment:
Could we not use state.startKey here? Is that not a thing?

Reply:
It is a thing, finishing up some testing and then will push